.github/workflows/main.yml

```diff
@@ -0,0 +1,38 @@
+name: Build and Deploy
+on:
+  push:
+    branches:
+      - main
+      - feature/#25-hosting
+jobs:
+  build-and-deploy:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout 🛎️
+        uses: actions/checkout@v2.3.1
+
+      - uses: lukka/get-cmake@latest
+
+      - name: Setup emsdk
+        uses: mymindstorm/setup-emsdk@v9
+        with:
+          version: 2.0.16
+
+      #Following two just for info in case something goes wrong
+      - name: Verify
+        run: emcc -v
+
+      - name: Tree
+        run: tree
+
+      - name: Run emCMake
+        run: |
+          cd demo/backend/
+          emcmake cmake
+          make install
```

Comment:
According to the [docs](https://emscripten.org/docs/compiling/Building-Projects.html) `make` should also be wrapped. Obviously it builds as it is but this should be more fail safe.